clients/java/signalr/build.gradle

```diff
@@ -54,8 +56,7 @@ task generatePOM {
                 }
             }
         }
-    }.writeTo("signalr-0.1.0-preview1.pom")
-
-    ant.move file: "signalr-0.1.0-preview1.pom",
-            todir: "${buildDir}/libs"
+    }.writeTo("${buildDir}/libs/signalr-${project.version}.pom")
```

Comment:
This is better 👍

Reply:
Yeah, it was causing the POM file to appear even when just running the `compileJava` task. Not sure why but removing the `ant.move` and having the POM file written directly to the right place fixed it.